cmds/dump.c

```diff
@@ -544,10 +548,13 @@ static void dump_raw_header(struct uftrace_dump_ops *ops, struct uftrace_data *h
 	       handle->hdr.endian == 1 ? "little" : "big");
 	pr_out("uftrace file header: class         = %u (%s bit)\n", handle->hdr.elf_class,
 	       handle->hdr.elf_class == 2 ? "64" : "32");
-	get_feature_string(buf, sizeof(buf), handle->hdr.feat_mask);
+	get_header_string(buf, sizeof(buf), handle->hdr.feat_mask, feat_str, FEAT_BIT_MAX);
 	pr_out("uftrace file header: features      = %#" PRIx64 " (%s)\n", handle->hdr.feat_mask,
 	       buf);
-	pr_out("uftrace file header: info          = %#" PRIx64 "\n", handle->hdr.info_mask);
+	memset(buf, 0, sizeof(buf));
```

Comment:
Do we need this `memset`?

Reply:
I thought that `buf` needs to be cleared if feature name length is shorter than info name length.
But `snprintf` includes the terminating null byte ('\0')) to `buf`, so `memset` doesn't need.
Thanks for your comment :)